geode-wan/src/test/java/org/apache/geode/internal/cache/wan/parallel/ParallelWANConflationDUnitTest.java

```diff
@@ -108,24 +113,35 @@ public void testParallelPropagationBatchConflation() throws Exception {
       vm4.invoke(() -> putGivenKeyValue(getTestMethodName(), keyValues));
     }
 
+    // sender did not turn on conflation, so queue size will be 100 (otherwise it will be 20)
```

Comment:
This test is named conflation, is there a reason why it is now getting 100 instead of 20?

Reply:
The test is for batch conflation, and purposely not to turn on the conflation during enqueue. So the queue size in this test will be 100.  

We have other cases to turn on conflation during enqueue, where the queue size will be 20.